packages/api/src/routes/medical/patient.ts

```diff
@@ -31,7 +31,6 @@ import {
   getPatientOrFail,
   getPatients,
   matchPatient,
-  PatientMatchCmd,
```

Comment:
We can probably remove this type completely right?

Reply:
Nah, it's used on `matchPatient()` and it passes as parameter to `sanitize()`.

Reply:
👍